evals/evaluation/crud_rag/crud_evaluator.py

```diff
@@ -0,0 +1,235 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
hi, xinyu, shall we make the implementations more general? we can also use your code to evaluate other datasets. some implementations related to crud dataset, I think we can move them into `examples`. we can discuss this offline.

Reply:
it's done.